### PR TITLE
Rename analyze.preflight_check as cluster.crash_test

### PR DESCRIPTION
**Pull request by liangxin1300**
Changes include:
- Remove analyze sub-level and rename preflight_check as cluster.crash_test
- Replace --yes as --force
```
# crm cluster crash_test 
usage: crash_test
                  [--kill-sbd | --kill-corosync | --kill-pacemakerd | --fence-node NODE | --split-brain-iptables]
                  [-l] [-f] [-h]

Cluster crash test tool set. It standardizes the steps to simulate
cluster failures and to verify some key configuration before you move
your cluster into production. It is carefully designed with the proper
steps and does not change any configuration to harm the cluster without
the confirmation from users.

optional arguments:
  --kill-sbd              Kill sbd daemon
  --kill-corosync         Kill corosync daemon
  --kill-pacemakerd       Kill pacemakerd daemon
  --fence-node NODE       Fence specific node
  --split-brain-iptables  Make split brain by blocking corosync ports
  -l, --kill-loop         Kill process in loop

other options:
  -f, --force             Force to skip all prompts (Use with caution, the
                          intended fault will be injected to verify the
                          cluster resilence)
  -h, --help              Show this help message and exit

Log: /var/log/crmsh/crash_test.log
Json results: /var/lib/crmsh/crash_test/crash_test.json
For each --kill-* testcase, report directory: /var/lib/crmsh/crash_test
```
- Add big warnings to have users' attention to those operation might introduce failover
```
# crm cluster crash_test --kill-corosync

==============================================
Testcase:          Force kill corosync
Looping Kill:      False
Expected State:    a) corosync process restarted
                   b) Or, this node fenced.

!!! WARNING WARNING WARNING !!!
THIS CASE MAY LEAD TO NODE BE FENCED.
TYPE Yes TO CONTINUE, OTHER INPUTS WILL CANCEL THIS CASE [Yes/No](No):
```
Screenshot from 2021-06-11 10-51-46

#### rpm for test
https://build.opensuse.org/package/show/home:XinLiang:branches:preflight_check/crmsh

### CODE REVIEW

File: crmsh/crash_test/main.py [hunk 1]
```diff
@@ -164,7 +164,13 @@ def parse_argument(context):
...
     """
     parser = argparse.ArgumentParser(prog=context.process_name,
-                                     description='Cluster preflight check tool set',
+                                     description="""
+Cluster crash test tool set.
+It standardizes the steps to simulate cluster failures and to
+verify some key configuration before you move your cluster into
+production. It is carefully designed with the proper steps and does not
+change any configuration to harm the cluster without the confirmation
+from users.""",
```
Comment (zzhou1) [inline crmsh/crash_test/main.py:173]: The paragrah need reformat, eg.
```
Cluster crash test tool set. It standardizes the steps to simulate
cluster failures and to verify some key configuration before you move
your cluster into production. It is carefully designed with the proper
steps and does not change any configuration to harm the cluster without
the confirmation from users.
```

**Review by zzhou1 — COMMENTED** [review]
I prefer to ammend the commit log description, something like,
from  `Dev: crash_test: Add warning for damage options`
to  `Dev: crash_test: Add big warnings to have users' attention to potential failover`

What's your view about my comments in pr#807?

File: crmsh/crash_test/main.py (hunk 1 above)
Comment (liangxin1300) [reply crmsh/crash_test/main.py:173]: OK
Changed

**Review by zzhou1 — APPROVED** [review]

**Review by gao-yan — APPROVED** [review]
I'm personally a little hesitant about the fact that the other commands under `crm cluster` are used for the normal purposes of configuration/management, while this test command is a very special one. OTOH, I'm not sure either if it's good to keep it in a separate directive/category alone (at least for now) despite whether the naming of the directive "analyze" is inappropriate...

Any ways, I'll leave it to you since it's your decision to make :-)

**Pull request merged**